include/zephyr/sys/util.h

```diff
@@ -634,11 +634,6 @@ char *utf8_lcpy(char *dst, const char *src, size_t n);
 }
 #endif
 
-/* This file must be included at the end of the !_ASMLANGUAGE guard.
- * It depends on macros defined in this file above which cannot be forward declared.
- */
-#include <zephyr/sys/time_units.h>
-
```

Comment:
This particular hunk is an exact revert of fbf851cdc4d8f0a so the commit message deserves a `Fixes: commit fbf851cdc4d8f0a`

Reply:
Cool, done